src/world/message_server.cpp

```diff
@@ -308,6 +296,46 @@ void message_server_listen(const bool& requestExit)
     }
 }
 
+void cache_zone_settings()
+{
+    const char* query = "SELECT zoneid, zoneip, zoneport, misc FROM zone_settings;";
+    int         ret   = sql->Query(query);
+
+    if (ret == SQL_ERROR)
+    {
+        ShowError("Error loading zone settings from DB. World server will not send messages.");
```

Comment:
If this kills the world server, I'd upgrade this to ShowFatal, or ShowException (if that exists?), log very loudly, and then actually throw a std::runtime_error with a helpful message

Reply:
Done and thrown